### ISSUE TITLE
pubspec has a path dependency

### ISSUE THREAD

**Issue by sethladd** [bwu-dart/pubs#3]
Did you mean to use https://github.com/dart-lang/package_config ?

**Comment by zoechi** [comment]
Yup, checked in with path instead of git, I'll fix it immediately.

**Comment by zoechi** [comment]
Didn't publish (pub and GitHub) after the last update. Should be fixed now.
